projects/knora-ui/src/lib/viewer/operations/display-edit/display-edit.component.scss

```diff
@@ -101,6 +101,6 @@
 .edit .mat-icon,
 .save .mat-icon,
 .cancel .mat-icon {
-    width: 18px;
-    height: 18px;
-}
\ No newline at end of file
+    width: 16px;
+    height: 16px;
```

Comment:
According to google's [material design docs](https://google.github.io/material-design-icons/#styling-icons-in-material-design), it's not recommended to size an icon any smaller than 18px. They are just guidelines though and to me 16px isn't a huge difference from 18px. I would imagine 18px is the recommended smallest size because anything smaller will be hard to tap on a smartphone.

Reply:
ok thanks for your comment, I will reverse my changes about the sizing and merge the PR